doc/reference/algorithms/centrality.rst

```diff
@@ -105,3 +105,10 @@ Percolation
    :toctree: generated/
 
    percolation_centrality
+
+Second Order Centrality
+--------
```

Comment:
This line needs to be the same length as the "section title" line above it.  So in this case quite  bit more minus signs.